contrib/docker/Dockerfile

```diff
@@ -19,10 +19,10 @@ RUN cd /root \
         && rm fava/docs/changelog.rst \
         && python3 -mpip install ./fava \
         && python3 -mpip uninstall --yes pip \
-        && find /usr/local/lib/python3.6/site-packages -name *.so -print0|xargs -0 strip -v \
+        && find /usr/local/lib/python3.6/site-packages -name '*.so' -print0|xargs -0 strip -v \
+        && find /usr/local/lib/python3.6/site-packages -name '*.pyc' -delete \
```

Comment:
This was intentionally removed - don't delete Python bytecode

Reply:
Sorry I didn't notice it was previously removed. Can I ask why? The bytecode is regenerated on first execution, deleting upfront reduces the size of the image thats hanging around and duplicated multiple places by a fair bit at cost of slightly slower startup.